package.json

```diff
@@ -32,6 +32,7 @@
         "ag-grid-react": "^31.1.1",
         "core-js": "^3.6.4",
         "notistack": "^3.0.0",
+        "oidc-client": "^1.10.1",
```

Comment:
Isn't it a commons-ui dependency?

Reply:
But we import it directly in the the project so we need to add it in the deps